awsclients/accessanalyzer_clients.go

```diff
@@ -11,58 +11,58 @@ import (
 
 type AccessAnalyzerClient interface {
 	CreateAnalyzer(ctx workflow.Context, input *accessanalyzer.CreateAnalyzerInput) (*accessanalyzer.CreateAnalyzerOutput, error)
-	CreateAnalyzerAsync(ctx workflow.Context, input *accessanalyzer.CreateAnalyzerInput) *AccessanalyzerCreateAnalyzerResult
+	CreateAnalyzerAsync(ctx workflow.Context, input *accessanalyzer.CreateAnalyzerInput) *AccessanalyzerCreateAnalyzerFuture
```

Comment:
Feels like it should be `AccessAnalyzerCreateAnalyzerFuture` instead of `AccessanalyzerCreateAnalyzerFuture`

See comment on the template.

Reply:
Also, we shouldn't need a prefix with https://github.com/temporalio/temporal-aws-sdk-go/issues/6